src/include/misc.i

```diff
@@ -30,6 +30,22 @@ __wt_strdup(WT_SESSION_IMPL *session, const char *str, void *retp)
 }
 
 /*
+ * __wt_seconds --
```

Comment:
Is this used in a performance path? I preferred it being bundled in with the other timer functions.